spec/lib/disability_compensation/providers/rated_disabilities/evss_rated_disabilities_provider_spec.rb

```diff
@@ -1,11 +1,45 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require 'disability_compensation/factories/api_provider_factory'
 require 'disability_compensation/providers/rated_disabilities/evss_rated_disabilities_provider'
 require 'support/disability_compensation_form/shared_examples/rated_disabilities_provider'
 
 RSpec.describe EvssRatedDisabilitiesProvider do
-  let(:current_user) { build(:disabilities_compensation_user) }
+  let(:current_user) do
+    create(:evss_user)
+  end
+
+  let(:auth_headers) do
+    EVSS::AuthHeaders.new(current_user).to_h
+  end
+
+  before do
+    Flipper.disable(ApiProviderFactory::FEATURE_TOGGLE_RATED_DISABILITIES)
+    allow_any_instance_of(Auth::ClientCredentials::Service).to receive(:get_token).and_return('blahblech')
+  end
+
+  after do
+    Flipper.disable(ApiProviderFactory::FEATURE_TOGGLE_RATED_DISABILITIES)
```

Comment:
seems a little odd we're disabling the same feature flag before and after this runs. Just flagging to make sure that was intentional.

Reply:
this is intentional.

the way vets-api has Flipper set up is to disable by default in all environments _**except for**_ `test`.

we need to explicitly set these to test the appropriate and expected behavior :smile:

Reply:
also, the tests do not reset the feature flags, so, locally, on subsequent runs, if there is a feature flag disabled, the database does not set it back to enabled and vice versa.

the only option is to ensure that it is set the way we need it in the test.